docs/docs/platform/serverless-functions/index.mdx

```diff
@@ -0,0 +1,103 @@
+
+Serverless functions can be used to handle [event triggers](/platform/database/event-triggers), form submission, integrations (e.g. Stripe, Slack, etc), and more.
+
+---
+
+## Creating a Serverless Function
+
+Every `.js` (JavaScript) and `.ts` (TypeScript) file in the `functions/` folder of your Nhost app is its own serverless function.
+
+<Tabs groupId="language">
+  <TabItem value="ts" label="TypeScript" default>
+
+```ts title=functions/test.ts
+import { Request, Response } from 'express'
+
+export default (req: Request, res: Response) => {
+  res.status(200).send(`Hello ${req.query.name}!`)
+}
+```
+
+:::info
+
+You **MUST** install `express` locally in the base directory of your Nhost app.
+
+```bash
+npm install -d express @types/express
+# or yarn
+yarn add -d express @types/express
+```
+
+:::
+
+  </TabItem>
+  <TabItem value="js" label="JavaScript">
+
+```js title=functions/test.js
+export default (req, res) => {
+  res.status(200).send(`Hello ${req.query.name}!`)
+}
+```
+
+:::info
+
+You **MUST** install `express` locally in the base directory of your Nhost app.
+
+```bash
+npm install -d express
+# or yarn
+yarn add -d express
+```
+
+:::
+
+  </TabItem>
+</Tabs>
+
+## Deploying Serverless Function
+
+Serverless functions are automatically deployed using Nhost's [GitHub integration](/platform/github-integration).
+
+## Routing
+
+HTTP endpoints are automatically generated based on the file structure inside `functions/`.
+
+Here's an example of four serverless functions with their files and their HTTP endpoints:
+
+| File                        | HTTP Endpoint                                                 |
+| --------------------------- | ------------------------------------------------------------- |
+| `functions/index.js`        | `https://[app-subdomain].nhost.run/v1/functions/`             |
+| `functions/users/index.ts`  | `https://[app-subdomain].nhost.run/v1/functions/users`        |
+| `functions/users/active.ts` | `https://[app-subdomain].nhost.run/v1/functions/users/active` |
+| `functions/my-copmany.js`   | `https://[app-subdomain].nhost.run/v1/functions/my-company`   |
+
+---
+
+## Environment Variables
+
+[Environment variables](/platform/environment-variables) are available inside your serverless functions. Both in production and when running Nhost locally using the [Nhost CLI](/platform/cli).
+
+## Billing
+
+Serverless functions are billed per GB-sec or GB-hour. 1 GB-hour is 3600 GB-seconds.
+
+1 GB-sec is 1 function with 1 GB of RAM running for 1 second. If 1 function with 1 GB of RAM runs for 3600 seconds, that's the equivalent of 1 GB-hour.
+
+All serverless functions in production are running with 128 MB of RAM. It's not currently possible to change this.
+
+## Node Version
+
+All serverless functions in production are running Node.js version 14.x. It's not currently possible to change this.
```

Comment:
`t's not currently possible to change this.` -> I would remove, so users can ask for it